templates/otp.service.j2

```diff
@@ -0,0 +1,16 @@
+## NOTICE: This file is written by ansible, and any changes made here will be overwritten on
+#          next provision.
+#          Modify azavea.opentripplanner/templates/otp.conf.j2 to make changes stick.
+
+[Unit]
+Description=Start OpenTripPlanner process
+After={{otp_service_after}}
```

Comment:
Consistency nitpick, but it would be good to stick to a space between curly brace and variable name.